bench/wtperf/wtperf_run_py/wtperf_run.py

```diff
@@ -48,10 +48,11 @@
 test_stats_file = 'test.stat'
 
 
-def create_test_home_path(home: str, test_run: int, operation: str):
+def create_test_home_path(home: str, test_run: int, operations: list = None):
```

Comment:
Minor: With `from typing import List` we can annotate these as `operations: List[str]`

Reply:
Good suggestion - I'll make the change.